src/converter/renderers/render-aem.js

```diff
@@ -106,8 +107,11 @@ function transformHTML(htmlString, aemAuthorUrl, path) {
     path.includes('/perspectives/') &&
     !path.includes('/perspectives/authors')
   ) {
+    const pagePath = path.substring(path.lastIndexOf('/') + 1);
```

Comment:
@ruhisingh1 - with the way they name pages, i think this is fine but let's confirm with POs if we need to include at least the /perspectives/ folder in path.

Reply:
Done.